### ISSUE TITLE
Add appropriate metadata for linking and sharing Together

### ISSUE THREAD

**Issue by intelagense** [Caleb-Cohen/Together#227]
### Intro
This issue has been opened specifically to give 100Devs members a chance to get involved. The issues presented will have more instruction and be easier to solve. If you are already contributing to this project or your skills are well-ahead of this issue, consider letting someone else take this on!

## Summary

![image](https://user-images.githubusercontent.com/18331781/210231172-c86cb654-d7e5-494f-8f2d-a7bcc8c15c0f.png)

👆 There is currently... no metadata that describes the app when sharing the link. You will need to edit `Together/client/public/index.html` to add a short description and image that represents the app. 
> 💡 Seek feedback from the [#Together thread (100Devs Discord)](https://discord.com/channels/735923219315425401/1038482732633825442)

### Creating your branch
- Fork the Together repository.
- Navigate to the development branch using `git checkout development`.
- Create a new branch using `git branch -b your-branch-name-here`.
- If you have questions, more instructions about this can be found in our [contributing document.](https://github.com/CalebCohen/Together/blob/development/.github/CONTRIBUTING.md)

### Setting up a local server
- Execute `npm install`
- Execute `npm run dev-concurrent` in a vscode terminal.
- If you get errors, you may have to do one of the following:
- Delete your node_modules folder and then run `npm i`
- Make sure you have a .env file with the [necessary secrets](https://github.com/Caleb-Cohen/Together/wiki/02---Onboarding)

### What you need to do

1. Navigate to `Together/client/public/index.html`
2. Edit the meta description tag with *enticing* text.
3. Take a screenshot or source part of the image from the design discussion #17 (something that best represents the app)
4. Place the screenshot file somewhere in the public folder and link to it from index.html

Example code for linking image with meta properties:
```html
<meta property="og:image" content="url"/>
<meta property="og:image:secure_url" content="url"/>
```

### When you are done
-Use `git add .` and then `git commit -m 'add a descriptive commit message here'` to commit your changes to your local branch.
-Push your commit(s) using `git push`.
-Open your fork in GitHub and you will see a message saying you can create a pull request. Make sure your pull request is pulling into the 'development' branch instead of 'main'.

### Who to contact
If you have any questions, you can either send a message in the Together channel in 100Devs Discord.

### Relevant file locations
`Together/client/public/index.html`

## Potential Problems
You will need to do some research on your own to determine current best practices.

*If you're interested in working on this feature, please comment so you can be assigned*

**Comment by uaravindshenoy** [comment]
I'd like to take this up.

**Comment by Caleb-Cohen** [comment]
Hey there @uaravindshenoy just checking in on this! Thank you.

**Comment by uaravindshenoy** [comment]
Have been caught up in some personal stuff for a while, I'll try and get on this over the weekend if that's OK.

**Comment by Caleb-Cohen** [comment]
> Have been caught up in some personal stuff for a while, I'll try and get on this over the weekend if that's OK.

Yep that's ok! Please let us know if you expect any more delays.

**Comment by Letsbuilditnow** [comment]
Hey Caleb, I would like to take up this issue. If it's ok.

**Comment by Letsbuilditnow** [comment]
Hey Caleb, I would like to work on this issue.

**Comment by Caleb-Cohen** [comment]
> Hey Caleb, I would like to work on this issue.

assigned. @intelagense can you ensure that the issue is correct and @Letsbuilditnow doesn't re-do any of the work already completed?

**Comment by intelagense** [comment]
Here is another resource as well that I bookmarked when creating the issue:
https://web.dev/learn/html/metadata/

**Comment by Caleb-Cohen** [comment]
Hey there @itisMilan just checking in on how things are going!

**Comment by itisMilan** [comment]
Hey @Caleb-Cohen, I have added the meta tags but when I run the test I am getting some error related to prettier.  

```
11:4   error  Delete `␍`  prettier/prettier
  12:65  error  Delete `␍`  prettier/prettier
  13:36  error  Delete `␍`  prettier/prettier  14:25  error  Delete `␍`  prettier/prettier  15:16  error  Delete `␍`  prettier/prettier
  16:3   error  Delete `␍`  prettier/prettier
  17:1   error  Delete `␍`  prettier/prettier
  18:28  error  Delete `␍`  prettier/prettier

✖ 5327 problems (5327 errors, 0 warnings)
  5327 errors and 0 warnings potentially fixable with the `--fix` option.  
```

**Comment by timmyichen** [comment]
I thought git automatically fixes these for you?  And why are there _five thousand_ of them 👀

**Comment by Caleb-Cohen** [comment]
> I thought git automatically fixes these for you? And why are there _five thousand_ of them 👀

Prettier is really starting to irk me, why is it so hard to get it to be consistent between developer environments?

**Comment by code2244** [comment]
> > Hey @Caleb-Cohen, I have added the meta tags but when I run the test, I am getting some error related to prettier.
> > ```
> > 11:4   error  Delete `␍`  prettier/prettier
> >   12:65  error  Delete `␍`  prettier/prettier
> >   13:36  error  Delete `␍`  prettier/prettier  14:25  error  Delete `␍`  prettier/prettier  15:16  error  Delete `␍`  prettier/prettier
> >   16:3   error  Delete `␍`  prettier/prettier
> >   17:1   error  Delete `␍`  prettier/prettier
> >   18:28  error  Delete `␍`  prettier/prettier
> > 
> > ✖ 5327 problems (5327 errors, 0 warnings)
> >   5327 errors and 0 warnings potentially fixable with the `--fix` option.  
> > ```
> 
> 👆 @timmyichen @RascalTwo I know this is a CRLF vs LF problem. Do y'all have a best practice answer for how to correct this or avoid it entirely in the future?

I think this issue is related to this recent PR . https://github.com/Together-100Devs/Together/pull/369

**Comment by Caleb-Cohen** [comment]
@itisMilan 

can you attempt changing [CRLF to LF](https://cdn.discordapp.com/attachments/1038482732633825442/1083362390613176350/image.png)?

**Comment by itisMilan** [comment]
> @itisMilan
> 
> can you attempt changing [CRLF to LF](https://cdn.discordapp.com/attachments/1038482732633825442/1083362390613176350/image.png)?

@intelagense suggested it and I have done that but still getting the same error.

**Comment by Caleb-Cohen** [comment]
> > @itisMilan
> 
> > 
> 
> > can you attempt changing [CRLF to LF](https://cdn.discordapp.com/attachments/1038482732633825442/1083362390613176350/image.png)?
> 
> 
> 
> @intelagense suggested it and I have done that but still getting the same error.

Brutal. I noticed you have to save the file after changing the setting. Are you sure you saved the setting?

**Comment by itisMilan** [comment]
Yes, I have saved it. So this error comes when I run "npm run lint" but, when I run "npm run test-frontend" it's showing "all specs passed!"

**Comment by itisMilan** [comment]
Updated the endOfLine to auto in .prettierrc file, now the errors are gone.
